NAMESPACE

```diff
@@ -30,6 +30,13 @@ S3method(format,DescriptionAuthorsAtR)
 S3method(format,DescriptionCollate)
 S3method(format,DescriptionDependencyList)
 S3method(format,DescriptionField)
+S3method(to_latex,DescriptionAuthorsAtR)
+S3method(to_latex,DescriptionCollate)
+S3method(to_latex,DescriptionField)
+S3method(to_latex,DescriptionURL)
+S3method(to_latex,DescriptionURLList)
+S3method(to_latex,character)
+S3method(to_latex,person)
```

Comment:
Hadley once said this is necessary, and I find evidence in R-exts, section 1.5.2:

> The standard method for S3-style UseMethod dispatching might fail to locate methods defined in a package that is imported but not attached to the search path. To ensure that these methods are available the packages defining the methods should ensure that the generics are imported and register the methods using S3method directives. If a package defines a function print.foo intended to be used as a print method for class foo, then the directive

>     S3method(print, foo)

> ensures that the method is registered and available for UseMethod dispatch, and the function print.foo does not need to be exported. Since the generic print is defined in base it does not need to be imported explicitly.